### ISSUE TITLE
Quarkus/Vertx WebClient return a `java.net.ConnectException: Connection refused` over FIPS

### ISSUE THREAD

**Issue by pjgg** [quarkusio/quarkus#23865]
### Describe the bug

When I make an HTTP request in a `@QuarkusTest` over RHEL8.5 + FIPS I am getting the following error:

```
2022-02-22 09:32:37,583 ERROR [io.qua.mut.run.MutinyInfrastructure] (vert.x-eventloop-thread-1) Mutiny had to drop the following exception: io.netty.channel.AbstractChannel$AnnotatedConnectException: Connection refused: localhost/127.0.0.1:8081
Caused by: java.net.ConnectException: Connection refused
	at java.base/sun.nio.ch.SocketChannelImpl.checkConnect(Native Method)
	at java.base/sun.nio.ch.SocketChannelImpl.finishConnect(SocketChannelImpl.java:779)
	at io.netty.channel.socket.nio.NioSocketChannel.doFinishConnect(NioSocketChannel.java:330)
	at io.netty.channel.nio.AbstractNioChannel$AbstractNioUnsafe.finishConnect(AbstractNioChannel.java:334)
	at io.netty.channel.nio.NioEventLoop.processSelectedKey(NioEventLoop.java:710)
	at io.netty.channel.nio.NioEventLoop.processSelectedKeysOptimized(NioEventLoop.java:658)
	at io.netty.channel.nio.NioEventLoop.processSelectedKeys(NioEventLoop.java:584)
	at io.netty.channel.nio.NioEventLoop.run(NioEventLoop.java:496)
	at io.netty.util.concurrent.SingleThreadEventExecutor$4.run(SingleThreadEventExecutor.java:986)
	at io.netty.util.internal.ThreadExecutorMap$2.run(ThreadExecutorMap.java:74)
	at io.netty.util.concurrent.FastThreadLocalRunnable.run(FastThreadLocalRunnable.java:30)
	at java.base/java.lang.Thread.run(Thread.java:834)
```

The same test succeed without FIPS enabled. 
Reproducer: 

`git@github.com](mailto:git@github.com):quarkus-qe/quarkus-test-suite.git`

`mvn clean verify -Dall-modules -pl http/vertx-web-client -Dit.test=PureVertxHttpClientTest`

Be sure that test is disabled

### Output of `java -version`

openjdk 11.0.11 2021-04-20 OpenJDK Runtime Environment AdoptOpenJDK-11.0.11+9 (build 11.0.11+9) OpenJDK 64-Bit Server VM AdoptOpenJDK-11.0.11+9 (build 11.0.11+9, mixed mode)

### Quarkus version or git rev

Quarkus 2.7.1.Final

**Comment by sberyozkin** [comment]
@pjgg Can you experiment with https://quarkus.io/guides/security-customization#bouncy-castle-jsse-fips please ? See also https://github.com/quarkusio/quarkus/tree/main/integration-tests/bouncycastle-fips-jsse

**Comment by pjgg** [comment]
This issue could be closed because is working on Quarkus 2.7.5 over FIPS
```
sysctl crypto.fips_enabled
crypto.fips_enabled = 1
```

cmd:
```
mvn clean verify -Dall-modules -pl http/vertx-web-client -Dit.test=PureVertxHttpClientTest
Tests run: 1, Failures: 0, Errors: 0, Skipped: 0
```
